examples/mnist.py

```diff
@@ -23,15 +33,12 @@
 
 
 def _load_data(url=DATA_URL, filename=DATA_FILENAME):
-    try:
-        with gzip.open(filename, 'rb') as f:
-            data = pickle.load(f)
-    except (IOError, EOFError):
+    if not os.path.exists(filename):
```

Comment:
With incomplete downloads, the user will now be greeted with something like:
``IOError: CRC check failed 0x15161f5c != 0x9ad4c9afL``

Seems like a friendly enough error to me.